core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -55,6 +54,25 @@ object Http4sOrgPlugin extends AutoPlugin {
           name = Some("Check unused compile dependencies")),
         WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
         WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
+      ),
+      githubWorkflowBuildMatrixFailFast := Some(false),
+      githubWorkflowArtifactUpload := false,
```

Comment:
I don't have a firm grasp of what this one is supposed to practically achieve, or what problem it's causing.

Reply:
This feature exists to reuse the `target` directories from the build job in the publish job. In both http4s (see [here](https://github.com/http4s/http4s/blob/24d065b8a23f6f28bc7c0728575c7de95a159276/project/Http4sPlugin.scala#L274-L275)) and http4s-jdk-http-client, it tried to compress directories which do not exist. Maybe there is a workaround, but I don't think disabling this feature is problematic (and it can always be reenabled in downstream projects).